common/config/azure-pipelines/templates/core-build.yaml

```diff
@@ -34,7 +34,11 @@ steps:
     condition: and(succeeded(), eq(variables['Build.Reason'], 'PullRequest'), eq(variables['Agent.OS'], 'Darwin'))
   - script: npm run build:ios
     workingDirectory: "test-apps/display-test-app"
-    displayName: "Build iOS display-test-app"
+    displayName: "Build iOS display-test-app typescript"
+    condition: and(succeeded(), eq(variables['Build.Reason'], 'PullRequest'), eq(variables['Agent.OS'], 'Darwin'))
+  - script: npm run build:ios-app
```

Comment:
Do we need to configure an xcode profile or can we remove the one mentioned here? 

https://dev.azure.com/bentleycs/iModelTechnologies/_build/results?buildId=1647223&view=logs&j=bd4fcb09-6f48-5190-b780-f35cec3cbe42&t=5ae6fe72-8369-5bd3-c235-60b332798a2d&l=38

Reply:
Investigating...

Reply:
With the latest command-line args, the building and running for simulator works without any provisioning and signing.